### DISCUSSION TITLE
Trunk doesn't work in nested repo

### DISCUSSION THREAD

**Discussion by gewenyu99** [trunk-io/docs#282] [Feature Requests]
Originally by Duc Nguyen, moved from Canny:

I have a github repo a location a/ , I also have b/ inside of a/, thus a/b/ is the path I'm looking at, this make trunk stop working. When I move a/ and /b to separate location, trunk work again.

**Comment by Alexander-kniit** [comment]
Do you mean nested as submodules? If so, there is a feature request for that (https://github.com/orgs/trunk-io/discussions/307)
